build/kickstarts/base.ks.erb

```diff
@@ -259,4 +263,8 @@ rm -f /var/www/miq/vmdb/certs/v2_key
 
 [[ -s /etc/default/evm ]] && source /etc/default/evm
 appliance_console_cli --region 0 --internal --password smartvm
+
+pushd $app_root
+#copy opt dir from COPY
+cp -vr system/COPY/opt /
```

Comment:
Can this be moved up to the other copy so they live next to each other?  Or is that not possible.  If it's not possible, can you make a comment in the script here?